api/src/pcapi/routes/backoffice_v3/accounts.py

```diff
@@ -486,6 +467,21 @@ def _get_latest_fraud_check(
     return latest_fraud_check
 
 
+def _get_duplicate_fraud_history(
+    eligibility_history: dict[str, accounts.EligibilitySubscriptionHistoryModel],
+) -> str | None:
+    for history_item in eligibility_history.values():
+        for check_history in history_item.idCheckHistory:
+            reason_codes = set(check_history.reasonCodes) if check_history.reasonCodes else None
+            duplicate_user_code = fraud_models.FraudReasonCode.DUPLICATE_USER.value
```

Comment:
C'est une constante, donc : 
- ça n'apporte rien et complique la lisibilité de la remettre dans une variable
- c'est "bête" de l'affecter à chaque itération des boucles `for`